plenum/server/consensus/ordering_service.py

```diff
@@ -2075,3 +2070,49 @@ def l_processReqDuringBatch(self,
         if self.is_master:
             self.l_do_dynamic_validation(req, cons_time)
             self._write_manager.apply_request(req, cons_time)
+
+    def can_send_3pc_batch(self):
```

Comment:
1) I would prefer having this in the Validator class
2) Please copy the code as is. Keep usage of `pre_view_change_in_progress`.